x-pack/plugins/security_solution/cypress/e2e/exceptions/entry/flyout_validation.cy.ts

```diff
@@ -62,20 +62,24 @@ import {
 } from '../../../tasks/api_calls/exceptions';
 import { getExceptionList } from '../../../objects/exception';
 
+// Test Skipped until we fix the Flyout rerendering issue
+// https://github.com/elastic/kibana/issues/154994
+
 // NOTE: You might look at these tests and feel they're overkill,
 // but the exceptions flyout has a lot of logic making it difficult
 // to test in enzyme and very small changes can inadvertently add
 // bugs. As the complexity within the builder grows, these should
 // ensure the most basic logic holds.
-describe('Exceptions flyout', { testIsolation: false }, () => {
+describe.skip('Exceptions flyout', { testIsolation: false }, () => {
```

Comment:
NIT: Let's open a ticket to track this test is going to be skipped.

Reply:
In the PR description, I linked the needed ticket https://github.com/elastic/kibana/issues/154994  :)